data/Surfaces/abelian_d15_pi21_quintic_3

```diff
@@ -1 +1 @@
,"29631"],[["1","1","0","0","5"],"1"],[["1","0","3","2","1"],"1"],[["1","0","1","1","4"],"9533"],[["0","4","2","1","0"],"31990"],[["0","2","2","2","1"],"11996"],[["0","2","0","1","4"],"22114"],[["0","1","4","1","1"],"9533"],[["0","1","1","2","3"],"1"],[["0","0","6","0","1"],"22114"],[["0","0","3","1","3"],"11996"],[["0","0","2","3","2"],"9533"],[["0","0","1","5","1"],"22114"],[["0","0","1","0","6"],"22114"]],[[["4","0","2","0","1"],"22114"],[["3","0","1","1","2"],"1"],[["2","4","0","0","1"],"31990"],[["2","1","2","0","2"],"9533"],[["2","1","1","2","1"],"12237"],[["1","5","0","1","0"],"9877"],[["1","3","1","0","2"],"9877"],[["1","2","2","1","1"],"3"],[["1","1","4","0","1"],"11996"],[["1","1","0","3","2"],"1"],[["1","0","5","1","0"],"9877"],[["0","6","1","0","0"],"31990"],[["0","4","1","1","1"],"11996"],[["0","3","3","0","1"],"9533"],[["0","2","2","0","3"],"11996"],[["0","2","1","2","2"],"9533"],[["0","2","0","4","1"],"22114"],[["0","1","6","0","0"],"31990"],[["0","1","3","1","2"],"9877"],[["0","1","1","0","5"],"31990"],[["0","0","4","2","1"],"31990"]],[[["5","0","0","0","0"],"9877"],[["3","1","0","0","1"],"22458"],[["2","1","2","0","0"],"19995"],[["2","0","0","2","1"],"19995"],[["1","3","1","0","0"],"22458"],[["1","2","0","0","2"],"19995"],[["1","1","1","1","1"],"14597"],[["1","0","0","1","3"],"22458"],[["0","5","0","0","0"],"9877"],[["0","2","1","2","0"],"19995"],[["0","1","3","1","0"],"22458"],[["0","0","5","0","0"],"9877"],[["0","0","2","1","2"],"19995"],[["0","0","1","3","1"],"22458"],[["0","0","0","5","0"],"9877"],[["0","0","0","0","5"],"9877"]],[[["3","1","0","0","1"],"1"],[["2","2","0","1","0"],"22114"],[["2","0","1","0","2"],"22114"],[["1","3","1","0","0"],"1"],[["1","1","1","1","1"],"19995"],[["1","0","2","2","0"],"22114"],[["1","0","0","1","3"],"1"],[["0","2","2","0","1"],"22114"],[["0","1","3","1","0"],"1"],[["0","1","0","2","2"],"22114"],[["0","0","1","3","1"],"1"]],[[["3","0","1","1","0"],"9877"],[["2","1","2","0","0"],"31990"],[["2","0","0","2","1"],"31990"],[["1","2","0","0","2"],"31990"],[["1","1","1","1","1"],"22458"],[["1","1","0","3","0"],"9877"],[["1","0","3","0","1"],"9877"],[["0","3","0","1","1"],"9877"],[["0","2","1","2","0"],"31990"],[["0","1","1","0","3"],"9877"],[["0","0","2","1","2"],"31990"]],[[["5","0","0","0","0"],"31990"],[["3","0","1","1","0"],"11996"],[["2","2","0","1","0"],"9533"],[["2","0","1","0","2"],"9533"],[["1","1","1","1","1"],"5"],[["1","1","0","3","0"],"11996"],[["1","0","3","0","1"],"11996"],[["1","0","2","2","0"],"9533"],[["0","5","0","0","0"],"31990"],[["0","3","0","1","1"],"11996"],[["0","2","2","0","1"],"9533"],[["0","1","1","0","3"],"11996"],[["0","1","0","2","2"],"9533"],[["0","0","5","0","0"],"31990"],[["0","0","0","5","0"],"31990"],[["0","0","0","0","5"],"31990"]]],"refs":{"412b7d25-2fb3-488d-9818-5d2e0fbb55ce":{"type":"MPolyRing","data":{"base_ring":{"type":"Nemo.fpField","data":"31991"},"symbols":["x","y","z","u","v"]}}},"_ns":{"Oscar":["https://github.com/oscar-system/Oscar.jl","0.13.0-DEV"]}}
```

Comment:
Meta comment (not specific to this PR): it seems most of the type information comes after the data. But don't we need the type information before the data when loading?

Reply:
the refs come after the data. The type comes before data when there is a ref the load wll jump to that section.

Reply:
Let me remind you that a JSON object is a dictionary.  There is *no* ordering of the individual properties.